label_studio/ml_models/serializers.py

```diff
@@ -16,19 +17,103 @@ class CreatedByFromContext:
     def __call__(self, serializer_field):
         return serializer_field.context.get('created_by')
 
-
 class ModelInterfaceSerializer(serializers.ModelSerializer):
     created_by = UserSimpleSerializer(default=CreatedByFromContext(), help_text='User who created Dataset')
+ 
+    class Meta:
+        model = ModelInterface
+        fields = '__all__'
+        read_only_fields = ['created_by', 'created_at', 'updated_at']
+
+    def check_output_classes(self, project, provided_output_classes):
+        labels, _ = get_all_labels(project.label_config)
+        project_output_classes =  sorted(list(set([label for label_list in labels.values() for label in label_list])))
+        print(project_output_classes)
+        print(provided_output_classes)
+        if project_output_classes != provided_output_classes:
+            raise ValidationError(f'output_classes not compatible with Project (id:{project.pk})')
```

Comment:
Mind adding both the `provided_output_classes` and the `project_output_classes` to the error message here?

Reply:
sure